src/main/java/nwoolcan/view/model/DashBoard.java

```diff
@@ -0,0 +1,33 @@
+package nwoolcan.view.model;
+
+/**
+ * View-Model interface with methods callable to obtain general information about the warehouse.
+ * There will be information about:
+ * <ul>
+ *     <li>Number of {@link nwoolcan.model.brewery.production.batch.Batch} which are in progress;</li>
+ *     <li>{@link nwoolcan.model.brewery.warehouse.stock.Stock} which are going to expire;</li>
+ *     <li>{@link nwoolcan.model.brewery.warehouse.stock.Stock} which have a low {@link nwoolcan.model.utils.Quantity}.</li>
+ * </ul>
+ */
+public final class DashBoard {
```

Comment:
If is a view model should be called `DashBoardViewModel`

Reply:
Actually it does not represent any class of the model, but it will contain attributes from different classes of the model. this is the reason why it was put there. Is it standard to call viewModel also these type of classes? isn't it redundant if we call everything "viewModel" when it is all in its package?

Reply:
nope it is not redundant. this class is ofc a view model (an object for transferring data and representing data from the model to view)